### PR TITLE
New instrument cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
PR includes new instrument cal csvs for refurb'd instruments:
ADCPT-F SN 19221
OPTAA-D SNs 159, 205

### CODE REVIEW

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/ADCPTF/CGINS-ADCPTF-19221__20210615.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+serial,name,value,notes
+19221,CC_scale_factor1,0.45, source_file ADCPT-F_WHS150-1500_SN_19221_Configuration_Summary_2021-06-15.pdf
+19221,CC_scale_factor2,0.45,
+19221,CC_scale_factor3,0.45,
+19221,CC_scale_factor4,0.45,
```
Comment (snwhite) [inline calibration/ADCPTF/CGINS-ADCPTF-19221__20210615.csv:5]: Looks good.  All values are constants.

File: calibration/OPTAAD/CGINS-OPTAAD-00159__20210510.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+ACS-159,CC_tbins,"[0.483895, 1.396066, 2.451290, 3.440000, 4.507500, 5.501667, 6.506250, 7.502692, 8.501250, 9.500000, 10.490833, 11.487600, 12.490000, 13.492083, 14.504400, 15.481667, 16.479600, 17.505600, 18.495000, 19.498636, 20.533000, 21.508800, 22.508462, 23.502400, 24.493333, 25.484800, 26.490800, 27.465769, 28.502609, 29.538889, 30.481538, 31.496538, 32.496538, 33.515000, 34.506400]",
+ACS-159,CC_taarray,SheetRef:CC_taarray,
+ACS-159,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00159__20210510.csv:9]: Spot check looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_taarray.ext [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+0.014548,0.010139,0.00903,0.008583,0.007885,0.007565,0.007035,0.00661,0.005965,0.005617,0.005153,0.004683,0.004143,0.003887,0.003191,0.002739,0.002436,0.001876,0.0015,0.001232,0.000824,0.000504,0.000301,0.000209,0.000148,0,-1.2e-05,-0.000163,-0.00043,-0.000302,-0.000291,-0.000516,-0.000388,-0.000443,-0.000477
+0.014494,0.010088,0.00909,0.008633,0.007989,0.007597,0.007103,0.006516,0.005954,0.00556,0.004976,0.004572,0.004172,0.003862,0.003229,0.002617,0.00234,0.001894,0.00138,0.001161,0.000765,0.000537,0.000243,0.000154,0.000207,0,-9.8e-05,-0.000113,-0.000439,-0.000162,-0.000261,-0.000442,-0.000301,-0.000468,-0.000475
+0.014688,0.010199,0.009048,0.008758,0.00792,0.007601,0.007177,0.006665,0.006071,0.00564,0.00505,0.004553,0.004126,0.003716,0.003133,0.002585,0.00232,0.001839,0.001351,0.001117,0.000675,0.000477,0.000202,0.000123,8.7e-05,0,-0.000104,-0.000147,-0.000467,-0.000352,-0.000327,-0.000537,-0.000451,-0.000594,-0.000539
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_taarray.ext:87]: Spot check looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_tcarray.ext [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+0.003107,0.001805,0.001556,0.001313,0.001186,0.000936,0.00102,0.000867,0.000722,0.00065,0.000659,0.000726,0.000486,0.000363,0.000266,0.000369,0.000314,0.000296,0.000185,0.000175,0.000282,0.000101,0.000218,0.000193,0.000161,0,-0.000119,4.1e-05,7e-05,9.7e-05,0.000189,0.000109,-2.3e-05,0.000147,-7.3e-05
+0.003602,0.001959,0.001885,0.001641,0.001463,0.001227,0.001223,0.0012,0.001056,0.00106,0.000884,0.000964,0.000784,0.000645,0.000582,0.000558,0.000416,0.000452,0.000337,0.000248,0.000395,0.000295,9.9e-05,-2.5e-05,-1.1e-05,0,-6e-05,-8e-06,-0.000174,-0.000138,-0.000156,-0.000253,-0.000251,-0.000283,-0.000555
+0.004675,0.002957,0.002751,0.00231,0.002538,0.002136,0.001873,0.001754,0.001464,0.001537,0.001528,0.001348,0.001147,0.001068,0.001004,0.001044,0.000876,0.000769,0.000614,0.000644,0.000481,0.000334,0.000369,0.000298,0.000227,0,-4.8e-05,0.000121,0.00014,-3e-05,-3.9e-05,-8.8e-05,-0.000447,-0.000269,-0.000558
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_tcarray.ext:87]: Spot check looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20210607.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+ACS-205,CC_tbins,"[0.660110, 1.379242, 2.420690, 3.446364, 4.540588, 5.511818, 6.498846, 7.528400, 8.500417, 9.476800, 10.471250, 11.494000, 12.493333, 13.481667, 14.470000, 15.482400, 16.482400, 17.490000, 18.502083, 19.509615, 20.496400, 21.497308, 22.527500, 23.493200, 24.479231, 25.501154, 26.488400, 27.477600, 28.482609, 29.513077, 30.495517, 31.483333, 32.497037, 33.526667, 34.430000, 35.000000]",
+ACS-205,CC_taarray,SheetRef:CC_taarray,
+ACS-205,CC_tcarray,SheetRef:CC_tcarray,
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00205__20210607.csv:9]: Spot check looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_tcarray.ext [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+0.001539,0.001901,0.001829,0.001735,0.001746,0.001332,0.001543,0.001393,0.001492,0.001339,0.001519,0.001631,0.001781,0.00167,0.00165,0.001469,0.00123,0.001102,0.000944,0.000835,0.000669,0.000454,0.000398,0.000334,0.000192,0,-0.000198,-0.000484,-0.000452,-0.000843,-0.001017,-0.001286,-0.001467,-0.001854,-0.002111,-0.002084
+0.000952,0.00142,0.00133,0.001251,0.00116,0.00115,0.000987,0.000933,0.000823,0.000831,0.000852,0.000956,0.001146,0.001078,0.000986,0.000966,0.000945,0.000586,0.000512,0.000563,0.000352,0.000368,0.000292,0.000157,0.000223,0,-0.000328,-0.000312,-0.00056,-0.000812,-0.001079,-0.001416,-0.001598,-0.001782,-0.002078,-0.002454
+0.001204,0.001665,0.001615,0.001432,0.001387,0.001126,0.00117,0.001348,0.001209,0.001237,0.001301,0.001348,0.001485,0.001398,0.001348,0.001289,0.000999,0.000935,0.000846,0.000693,0.000567,0.000481,0.000322,0.000309,8.2e-05,0,-0.000254,-0.000435,-0.000558,-0.000794,-0.001115,-0.001475,-0.001656,-0.001942,-0.002146,-0.002202
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_tcarray.ext:86]: Spot check looks good.

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_taarray.ext [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+-0.00862,-0.007394,-0.006774,-0.006356,-0.006292,-0.005792,-0.005458,-0.00511,-0.004747,-0.004327,-0.003995,-0.003563,-0.003365,-0.003019,-0.002659,-0.002334,-0.002007,-0.001703,-0.001286,-0.001055,-0.000702,-0.000515,-0.000395,-0.000265,8e-06,0,0.000212,0.000362,0.000375,0.000585,0.000575,0.000733,0.000691,0.000657,0.000646,0.000614
+-0.008772,-0.007449,-0.006881,-0.006395,-0.006403,-0.005891,-0.00552,-0.005235,-0.004758,-0.004397,-0.004089,-0.003623,-0.003402,-0.003126,-0.002777,-0.002446,-0.002056,-0.001675,-0.001335,-0.001048,-0.000716,-0.00056,-0.000489,-0.000239,-1.7e-05,0,0.000247,0.000373,0.00036,0.000743,0.000646,0.000723,0.000676,0.000697,0.000652,0.000843
+-0.009034,-0.00767,-0.007064,-0.00664,-0.006521,-0.00604,-0.00569,-0.005291,-0.004869,-0.004554,-0.004137,-0.003819,-0.003499,-0.003156,-0.002814,-0.002516,-0.00213,-0.001724,-0.001321,-0.001085,-0.000665,-0.000522,-0.000408,-0.000267,-4.7e-05,0,0.000189,0.000406,0.000408,0.000702,0.000689,0.000845,0.00073,0.000763,0.00074,0.00033
```
Comment (snwhite) [inline calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_taarray.ext:86]: Spot check looks good.

**Review by snwhite — APPROVED** [review]

File: calibration/ADCPTF/CGINS-ADCPTF-19221__20210615.csv (hunk 1 above)
Comment (cdobs) [reply calibration/ADCPTF/CGINS-ADCPTF-19221__20210615.csv:5]: looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00159__20210510.csv (hunk 2 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00159__20210510.csv:9]: spot checked. looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_taarray.ext (hunk 3 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_taarray.ext:87]: spot checked. looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_tcarray.ext (hunk 4 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00159__20210510__CC_tcarray.ext:87]: spot checked. looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20210607.csv (hunk 5 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00205__20210607.csv:9]: spot checked. looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_taarray.ext (hunk 7 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_taarray.ext:86]: spot checked. looks good

File: calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_tcarray.ext (hunk 6 above)
Comment (cdobs) [reply calibration/OPTAAD/CGINS-OPTAAD-00205__20210607__CC_tcarray.ext:86]: spot checked. looks good

**Review by cdobs — APPROVED** [review]
Most files were spot checked, but everything looks good

**Pull request merged**